test/functoria/query/install.expected

```diff
@@ -1,7 +1,7 @@
 bin: [
-  "test/functoria/query/main.exe" {"noop"}
+  "main.exe" {"noop"}
```

Comment:
this is not great, but should be fixed once everything use dune and we don't have to worry with .install files anymore.